README.md

```diff
@@ -111,3 +111,38 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+Place the tool somewhere in your path, using either:
+* make install
```

Comment:
I haven't tried that, but I suspect it would work. I think you still need to download git-secrets before running the brew command, but I can talk to Mohs to see if we can duplicate the effects of a manual install in a script. (To try to do it from a fresh computer.)